common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4231,7 +4231,7 @@
   },
   "UNSET_NONEXISTENT_PROPERTIES" : {
     "message" : [
-      "Attempted to unset non-existent properties [<properties>] in table <table>."
+      "Attempted to unset non-existent properties [<properties>] in relation <relationId>."
```

Comment:
I want to reuse it in the `namespace` scene

Reply:
but namespace is not a relation... maybe just `table or namespace <name>`

Reply:
Yea, okay.